server/renderer.js

```diff
@@ -151,6 +175,7 @@ async function serverRenderAuthorised(req, res, next, htmlData, uiLang, bot) {
     // Dispatching languages change updates tags and sources.
     store.dispatch(settings.setUILanguage({ uiLang: cookieUILang }));
     store.dispatch(settings.setContentLanguages({ contentLanguages: cookieContentLanguages }));
+    store.dispatch(backendApi.util.invalidateTags(wholeSimpleMode));
```

Comment:
What does invalidateTags do?
whileSimpleMode is an array with one string...

Reply:
https://redux-toolkit.js.org/rtk-query/usage/automated-refetching